sklearn/metrics/_plot/roc_curve.py

```diff
@@ -7,7 +7,7 @@
 class RocCurveDisplay:
     """ROC Curve visualization.
 
-    It is recommend to use `sklearn.metrics.plot_roc_curve` to create a
+    It is recommend to use :func:`~sklearn.metrics.plot_roc_curve` to create a
```

Comment:
what does the tilde do?

Reply:
With tilde:

![Screen Shot 2019-08-22 at 4 30 58 PM](https://user-images.githubusercontent.com/5402633/63547690-b0e24e00-c4fa-11e9-944c-484ad0226f33.png)

Without tilde:

![Screen Shot 2019-08-22 at 4 34 00 PM](https://user-images.githubusercontent.com/5402633/63547706-b770c580-c4fa-11e9-9cea-dd54eb4c2339.png)